daemons/controld/controld_membership.c

```diff
@@ -225,10 +225,10 @@ search_conflicting_node_callback(xmlNode * msg, int call_id, int rc,
         node_xml = output;
 
     } else {
-        node_xml = pcmk__xml_first_child(output);
+        node_xml = pcmk__xe_first_child(output);
     }
 
-    for (; node_xml != NULL; node_xml = pcmk__xml_next(node_xml)) {
+    for (; node_xml != NULL; node_xml = pcmk__xe_next(node_xml)) {
```

Comment:
This one can use `first_named_child()` and `crm_next_same_xml()`. There are a few others like that in this commit, they could be in their own commit (and even more that aren't in this commit, mostly in unpack.c, but we can ignore those for now)